### ISSUE TITLE
Apply to a grant and put someone else as the applicant

### ISSUE THREAD

**Issue by lmkw** [NUARIG/competitions#838]
Currently you can't name someone else as the applicant when you first apply to a grant. You have to create the submission and then go back and add the person as an applicant. You should be able to do that up front, when first creating the submission. This should be able to handle multiple applicants.

Care will be needed for the dependency to destroy the applicant if the submission is never created. Potentially look at responses as a model for this dependency.

**Comment by mattbaumann1** [comment]
This ticket has been paused to explore #881 to add necessary frameworks to carry out this ticket.

**Comment by lmkw** [comment]
11/09/2021 on hold until we can install js bundling

**Comment by lmkw** [comment]
dependent on #881
